sklearn/metrics/pairwise.py

```diff
@@ -607,6 +609,41 @@ def _argmin_reduce(dist, start):
     return dist.argmin(axis=1)
 
 
+_VALID_METRICS = [
```

Comment:
If the goal was to move these up, I think we should remove the original declaration:

https://github.com/scikit-learn/scikit-learn/blob/f034f57b1ad7bc5a7a5dd342543cea30c85e74ff/sklearn/metrics/pairwise.py#L1744